_posts/2022-11-25-checking-in-on-dawson.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2022-11-25
+title: >
+  18F Checks In With the DAWSON Project at the US Tax Court
+authors:
+  - chris-holly
+  - mike-marcotte
+  - alex-soble
+  - julia-allen
+  - mark-meyer
+tags:
+  - 18f
+  - agency work
+  - case management system
+  - interview
+  - open source
+  - tax court
+excerpt: >
+  The United States Tax Court is an independent federal court that settles disputes between taxpayers and the Internal Revenue Service. Since the 1980s, the Court has used a modified commercial software product to manage its operations. Last year the Court introduced DAWSON, a modern open-source case management system developed with assistance from an industry partner and 18F. DAWSON streamlined Court operations and changed the way the public interacts with the Court.
+---
+
+![The USTC DAWSON electronic case file system's homepage]({{ site.baseurl }}/assets/blog/ustc-dawson-checkin/dawson-home.png)
+
+The United States Tax Court is an independent federal court that settles disputes between taxpayers and the Internal Revenue Service. Since the 1980s, the Court has used a modified commercial software product to manage its operations. Last year the Court introduced [DAWSON](https://www.dawson.ustaxcourt.gov), a modern open-source case management system developed with assistance from an industry partner and 18F. DAWSON streamlined Court operations and changed the way the public interacts with the Court.
```

Comment:
```suggestion
The United States Tax Court is an independent federal court that settles disputes between taxpayers and the Internal Revenue Service. Since the 1980s, the Court has used a modified commercial software product to manage its operations. In December 2020 the Court introduced [DAWSON](https://www.dawson.ustaxcourt.gov), a modern open-source case management system developed with assistance from an industry partner and 18F. DAWSON streamlined Court operations and changed the way the public interacts with the Court.
```